src/components/Markdown/CodeBlock.tsx

```diff
@@ -29,7 +29,7 @@ export const CodeBlock: FC<{ children: React.ReactNode; language: string }> = ({
   };
 
   return (
-    <pre className="bg-cool-black text-white p-0 rounded-lg relative">
+    <pre className="bg-cool-black text-white p-0 rounded-lg relative max-w-[calc(100vw-48px)]">
```

Comment:
@kennethkalmer this is the thing that caps the page width

Reply:
@jamiehenson ooi would this need `dvw` instead of `vw`, or am I overeager to apply my new knowledge? 😃

Reply:
It's nifty isn't it. Such a clean solution. There's less of a need for it for width I would imagine (less funny business going on horizontally, only scrollbars really), but for height, yes. I think the best foot forward would be to lead with `dvh` and offer an equivalent `vh` class as well as a fallback - to appease our Firefox user

Reply:
I've included a fallback for the 100vh helper in ably-ui, but not doing anything here with the 100vws